e-moe/guzzle6-bundle/1.1/etc/packages/emoe_guzzle_bundle.yaml

```diff
@@ -0,0 +1,4 @@
+emoe_guzzle:
+    log:
+        enabled: true # Logging requests to Monolog
+        format: 'Guzzle: [{ts}] "{method} {uri} HTTP/{version}" {code}' # Optional log format customization
```

Comment:
I'm divided about this 🤔   I think both of you are right: 1) this is the default config, so why repeat it?; 2) it's nice to tell users how to customize this without forcing them to look for in the docs.